Taken from upstream commit aa0c92c014ec54bcf3eb933485f9ef245a66aee0
"Update Develocity hostname"

--- README.md.orig	2026-04-26 21:18:46
+++ README.md	2026-09-18 16:31:53
@@ -49,13 +49,13 @@
 
 ## Develocity
 
-[![Revved up by Develocity](https://img.shields.io/badge/Revved%20up%20by-Develocity-06A0CE?logo=Gradle&labelColor=02303A)](https://ge.junit.org/scans)
+[![Revved up by Develocity](https://img.shields.io/badge/Revved%20up%20by-Develocity-06A0CE?logo=Gradle&labelColor=02303A)](https://develocity.junit.org/scans)
 
 JUnit 5 utilizes [Develocity](https://gradle.com/) for [Build Scans](https://scans.gradle.com/),
 [Build Cache](https://docs.gradle.org/current/userguide/build_cache.html), and
 [Predictive Test Selection](https://docs.gradle.com/enterprise/predictive-test-selection/).
 
-The latest Build Scans are available on [ge.junit.org](https://ge.junit.org/). Currently,
+The latest Build Scans are available on [develocity.junit.org](https://develocity.junit.org/). Currently,
 only core team members can publish Build Scans on that server.
 You can, however, publish a Build Scan to [scans.gradle.com](https://scans.gradle.com/) by
 using the `--scan` parameter explicitly.
--- settings.gradle.kts.orig	2026-04-26 21:18:46
+++ settings.gradle.kts	2026-09-18 16:32:42
@@ -19,7 +19,7 @@
 }
 
 val buildParameters = the<BuildParametersExtension>()
-val develocityServer = "https://ge.junit.org"
+val develocityServer = "https://develocity.junit.org"
 val useDevelocityInstance = !gradle.startParameter.isBuildScan
 
 develocity {
--- gradle/plugins/build-parameters/build.gradle.kts.orig	2026-09-18 16:28:28
+++ gradle/plugins/build-parameters/build.gradle.kts	2026-09-18 16:30:21
@@ -33,7 +33,7 @@
 			group("buildCache") {
 				string("server") {
 					description =
-						"Remote build cache server address (protocol and hostname), e.g. https://eu-build-cache-ge.junit.org"
+						"Remote build cache server address (protocol and hostname), e.g. https://eu-develocity-node.junit.org"
 				}
 			}
 			group("predictiveTestSelection") {
--- gradle/plugins/common/src/main/kotlin/junitbuild.testing-conventions.gradle.kts.orig	2026-04-26 21:18:46
+++ gradle/plugins/common/src/main/kotlin/junitbuild.testing-conventions.gradle.kts	2026-09-18 16:27:47
@@ -96,7 +96,7 @@
 
 			// Ensure PTS works when publishing Build Scans to scans.gradle.com
 			this as PredictiveTestSelectionConfigurationInternal
-			server = uri("https://ge.junit.org")
+			server = uri("https://develocity.junit.org")
 
 			mergeCodeCoverage = true
 		}
